android/assets/jsons/Nations/Nations_Ukrainian.json

```diff
@@ -673,9 +673,6 @@
 			"Rakahanga","Bora Bora","Kailua","Uvea","Futuna","Rotuma","Tokelau","Lahaina","Bellona","Mungava","Tikopia",
 			"Emae","Kapingamarangi","Takuu","Nukuoro","Sikaiana","Anuta","Nuguria","Pileni","Nukumanu"]
 	},
-
-	/*
-
```

Comment:
The build is failing because you removed the comments. The block from *Aztecs* to *Nubia* should be commented out.